operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -113,113 +114,113 @@ Two very important notes regarding this integration:
 
-* **Happiness**: Delighters & requests/problems to solve coming from prospects or customers.
-* **Ops & Strategy**: Test infrastructure, technical debt, performance, maintenance, support cost reduction, packaging & pricing, feature discovery, branding, etc.
+* **Happiness:** Delighters & requests/problems to solve coming from prospects or customers.
+* **Ops & Strategy:** Test infrastructure, technical debt, performance, maintenance, support cost reduction, packaging and pricing, feature discovery, branding, etc.
 
-Each of these custom score columns has its own specific drivers that you can rate from 1-5. The more data you include here for each feature, the more accurate and useful the prioritization board will be. 
+Each of these custom score columns has its own specific drivers that you can rate from 1 - 5. The more data you include here for each feature, the more accurate and useful the prioritization board will be.
 
-### 1f - Publishing features in the internal portal
+### 1f Publishing features in the Internal portal
 
 To be added \(ETA May\).
 
-### 1g - Publishing features in the public portal
+### 1g Publishing features in the public portal
 
 To be added \(ETA Q2/FY21\)
 
-### 1h - Key links in ProductBoard
+### 1h Key links in productboard
 
-* [Insights Board](https://mattermost.productboard.com/insights)
-  * [Personal Inbox](https://mattermost.productboard.com/insights/my-inbox)
-  * [Shared Team Inbox](https://mattermost.productboard.com/insights/shared-inbox)
-* [Feature Board](https://mattermost.productboard.com/feature-board)
-  * [Master Feature List](https://mattermost.productboard.com/feature-board/1097524-master-feature-list)
-  * [Feature Organization View](https://mattermost.productboard.com/feature-board/1097524-feature-organization)
-  * [Top Requests](https://mattermost.productboard.com/feature-board/1097530-top-requests)
-  * [Prioritization Matrix](https://mattermost.productboard.com/feature-board/1097533-prioritization-matrix)
-* [ProductBoard Support](https://help.productboard.com/en/)
+* [Insights](https://mattermost.productboard.com/insights)
+  * [My inbox](https://mattermost.productboard.com/insights/my-inbox)
+  * [Shared inbox](https://mattermost.productboard.com/insights/shared-inbox)
+* [Features](https://mattermost.productboard.com/feature-board)
+  * [Master Feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list)
+  * [Feature organization view](https://mattermost.productboard.com/feature-board/1097524-feature-organization)
+  * [Top requests](https://mattermost.productboard.com/feature-board/1097530-top-requests)
```

Comment:
This isn't populated yet.

Reply:
Yes, we're not using this. we can remove this line.